content/resources/apt_package/_index.md

```diff
@@ -140,17 +140,10 @@ properties_list:
   required: false
   description_list:
   - markdown: The version of a package to be installed or upgraded.
-examples_list:
-- example_heading: Install a package using package manager
-  text_blocks:
-  - code_block: |
```

Comment:
Sorry that I'm coming late to this discussion. Did the literal block scalar just not work? I'm not wild about the hand curation needed in the solution.

Reply:
The issue is that all the examples need to be in one long MD string. Currently there are resource examples in `chef/chef`, but they're one long string and there's no easy way to get them from `chef/chef` to the current Hugo template. When Tim updates the resources using his rake task, then the MD will look readable not like the jumbled mess that it is here.

Reply:
Automate the world!